packages/block-editor/src/components/dimensions-tool/aspect-ratio-tool.js

```diff
@@ -117,7 +91,7 @@ export default function AspectRatioTool( {
 			<SelectControl
 				label={ __( 'Aspect ratio' ) }
 				value={ displayValue }
-				options={ options }
+				options={ aspectRatioOptions }
```

Comment:
Instead of merging `options` above, we can pass `options ?? aspectRatioOptions` and preserve the previous component behavior.

Reply:
That works :)

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/47271/commits/7f6223fe1c8f24eeed53b9e83a5697006201587e :)